tests/bootstrap.php

```diff
@@ -104,6 +104,10 @@
 
 ConnectionManager::setConfig('test', ['url' => getenv('DB_URL')]);
 
+if (env('CAKE_TEST_AUTOQUOTE')) {
+    ConnectionManager::get('test')->getDriver()->enableAutoQuoting(true);
+}
```

Comment:
You could also modify the setConfig() call on 105 to have the `quoteIdentifiers` option set. But this works as well.

Reply:
I just copied this from the old code.